gaia-header.js

```diff
@@ -123,6 +124,7 @@ module.exports = component.register('gaia-header', {
     debug('attached');
     this.runFontFitSoon();
     this.observerStart();
+    window.addEventListener('resize', this);
```

Comment:
But TBH I'm really not that bothered about if you want to stick with this approach :)